src/mpid/ch4/netmod/ofi/ofi_types.h

```diff
@@ -397,6 +350,23 @@ typedef struct {
     char kvsname[MPIDI_KVSAPPSTRLEN];
     char pname[MPI_MAX_PROCESSOR_NAME];
     int port_name_tag_mask[MPIR_MAX_CONTEXT_MASK];
+
+    /* Capability settings */
+#ifdef MPIDI_OFI_ENABLE_RUNTIME_CHECKS
```

Comment:
@wesbland: I don't see what changes you made.  The code seems to be the same.

Btw, shouldn't the max endpoints be a pointer rather than an array, so it can be set with CVARs as well?